server/embed/config_test.go

```diff
@@ -160,8 +160,20 @@ func (s *securityConfig) equals(t *transport.TLSInfo) bool {
 		s.ClientCertFile == t.ClientCertFile &&
 		s.ClientKeyFile == t.ClientKeyFile &&
 		s.KeyFile == t.KeyFile &&
-		s.AllowedCN == t.AllowedCN &&
-		s.AllowedHostname == t.AllowedHostname
+		compareSlices(s.AllowedCNs, t.AllowedCNs) &&
+		compareSlices(s.AllowedHostnames, t.AllowedHostnames)
```

Comment:
Suggestion to use [`slices.Equal`](https://pkg.go.dev/slices#Equal). However, this will drift from the original implementation.

Reply:
I think staying consistent seems to be a better option.

Reply:
Could you please help me to add `ok-to-test` label?

Reply:
Sure, now that you have addressed my other comment, let me trigger them.